doc/west/index.rst

```diff
@@ -1,14 +1,75 @@
 .. _west:
 
-West (Experimental)
-###################
+West
+####
 
-The Zephyr project is developing a swiss-army knife command line tool
+The Zephyr project maintains a swiss-army knife command line tool
 named ``west``. (Zephyr is an English name for the Greek god of the
 west wind.)
 
-West is developed in its own `repository on GitHub`_. A copy is
-currently maintained in Zephyr's :file:`scripts/meta/west` directory.
+West is developed in its own `repository on GitHub`_.
+
+West is used to obtain the source code for the Zephyr project and can also be
+used to build, debug and flash applications.
+
+.. _west-struct:
+
+Structure
+*********
+
+West is split into two distinct parts that work together to provide the
+full set of functionality offered by the tool:
+
+* Bootstrapper: Installed using ``pip``, implements ``west init``
+* Installation: Installed using ``west init``, implements all other commands
```

Comment:
I struggle with finding a good name for this but I'm not sure "installation" is right.

How about "main" west?

Reply:
I went over this several times and I thought this was the least bad one because of the following reasons:

* We are already using "installation" in some other parts of the doc, even before west
* It is unambiguous and includes both the fact that west is installed but also Zephyr itself in its term
* We might choose to add toolchain bits as part of the manifest in the future
* The other options have real drawbacks from my point of view:
  - "Worskpace" could suggest "view" in the model-view-controller metaphor, and this is more than just the view
  - "Main west" doesn't include the projects or manifest, just west itself. I am looking for a term that I can use both for the part of west that is cloned by `west init` but also that can be used generically to refer to the whole contents of the folder created with it

Let's discuss it a bit more.